packages/sdk/src/evm/core/classes/contract-royalty.ts

```diff
@@ -44,11 +44,13 @@ export class ContractRoyalty<
   }
 
   /**
-   * Gets the royalty recipient and BPS (basis points) of the contract
+   * Gets the royalty recipient and fee
```

Comment:
```suggestion
   * Get the royalty recipient and fee
```

I think we do "Get" instead of Gets everywhere?